<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>UITPad</class>
	<widget class="QMainWindow" name="UITPad">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>800</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>UITPad - Simple Text Editor</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<widget class="QTextEdit" name="textEdit">
						<property name="placeholderText">
							<string>Start typing your text here...</string>
						</property>
					</widget>
				</item>
			</layout>
		</widget>
		<widget class="QMenuBar" name="menuBar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>800</width>
					<height>22</height>
				</rect>
			</property>
			<widget class="QMenu" name="menuFile">
				<property name="title">
					<string>&amp;File</string>
				</property>
				<addaction name="actionOpen"/>
				<addaction name="actionSave"/>
				<addaction name="separator"/>
				<addaction name="actionExit"/>
			</widget>
			<widget class="QMenu" name="menuEdit">
				<property name="title">
					<string>&amp;Edit</string>
				</property>
				<addaction name="actionUndo"/>
				<addaction name="actionRedo"/>
				<addaction name="separator"/>
				<addaction name="actionCut"/>
				<addaction name="actionCopy"/>
				<addaction name="actionPaste"/>
			</widget>
			<widget class="QMenu" name="menuHelp">
				<property name="title">
					<string>&amp;Help</string>
				</property>
				<addaction name="actionAbout"/>
			</widget>
			<addaction name="menuFile"/>
			<addaction name="menuEdit"/>
			<addaction name="menuHelp"/>
		</widget>
		<action name="actionOpen">
			<property name="text">
				<string>&amp;Open</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+O</string>
			</property>
		</action>
		<action name="actionSave">
			<property name="text">
				<string>&amp;Save</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+S</string>
			</property>
		</action>
		<action name="actionExit">
			<property name="text">
				<string>E&amp;xit</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+Q</string>
			</property>
		</action>
		<action name="actionUndo">
			<property name="text">
				<string>&amp;Undo</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+Z</string>
			</property>
		</action>
		<action name="actionRedo">
			<property name="text">
				<string>&amp;Redo</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+Y</string>
			</property>
		</action>
		<action name="actionCut">
			<property name="text">
				<string>Cu&amp;t</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+X</string>
			</property>
		</action>
		<action name="actionCopy">
			<property name="text">
				<string>&amp;Copy</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+C</string>
			</property>
		</action>
		<action name="actionPaste">
			<property name="text">
				<string>&amp;Paste</string>
			</property>
			<property name="shortcut">
				<string>Ctrl+V</string>
			</property>
		</action>
		<action name="actionAbout">
			<property name="text">
				<string>&amp;About</string>
			</property>
		</action>
	</widget>
	<resources/>
	<connections/>
</ui>